### PR TITLE
feat: init eslint, prettier, cz-cli

### PR DESCRIPTION
**Pull request by niqqq25**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry=http://npm.intranet.1stdibs.com:4783/
```
Comment (twhid) [inline .npmrc]: We publish this module publicly, so we shouldn't point it at our private registry.

**Comment by niqqq25** [comment]
> This entire .npmrc file should be deleted.

I will delete it, when `dibs-prettier-config` is published on npm.

**Comment by robrichard** [comment]
I don't think we're going to publish `dibs-prettier-config`, so you can copy/paste it directly into this repo. 

I also don't think we should use cz/commitlint.

File: package.json [hunk 2]
```diff
@@ -39,7 +54,12 @@
         ]
     },
     "devDependencies": {
+        "eslint": "^6.1.0",
```
Comment (kostasmanionis) [inline package.json]: Make this `"eslint": "~6.1.0"` for now, https://github.com/eslint/eslint/issues/12117

**Pull request merged**